hack/lrp-test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Leader retry period (LRP) test
+# Iterate over LRP from 15ms to 500ms.
+# For each LRP value create a number of pods based on the number of nodes (N) and a capcity limit per node (C).
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+C=100 # capacity per node
+N=30  # number of compute nodes
```

Comment:
imo, it makes sense to have these as inputs, and default to these numbers. Your thoughts ?